plugins/main/public/components/overview/server-management-statistics/dashboards/dashboard_analysis_engine.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { getPlugins } from '../../../../kibana-services';
+import { ViewMode } from '../../../../../../../src/plugins/embeddable/public';
+import { I18nProvider } from '@osd/i18n/react';
+import useSearchBar from '../../../common/search-bar/use-search-bar';
+import { withErrorBoundary } from '../../../common/hocs/error-boundary/with-error-boundary';
+import { getDashboardPanelsAnalysisEngine } from './dashboard_panels_analysis_engine';
+import {
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiSelect,
+  EuiSpacer,
+  EuiCallOut,
+} from '@elastic/eui';
+import './statistics_dashboard.scss';
+import {
+  PatternDataSource,
+  tParsedIndexPattern,
+  useDataSource,
+} from '../../../common/data-source';
+import { IndexPattern } from '../../../../../../../src/plugins/data/public';
+import {
+  StatisticsDataSource,
+  StatisticsDataSourceRepository,
+} from '../../../common/data-source/pattern/statistics';
+import { LoadingSpinner } from '../../../common/loading-spinner/loading-spinner';
+
+const plugins = getPlugins();
+
+const SearchBar = getPlugins().data.ui.SearchBar;
+
+const DashboardByRenderer = plugins.dashboard.DashboardContainerByValueRenderer;
+
+interface DashboardStatisticsProps {
+  isClusterMode: boolean;
+  clusterNodes: any[];
+  clusterNodeSelected: any;
+  onSelectNode: any;
+}
+
+const DashboardStatistics: React.FC<DashboardStatisticsProps> = ({
+  isClusterMode,
+  clusterNodes,
+  clusterNodeSelected,
+  onSelectNode,
+}) => {
+  const {
+    filters,
+    fetchFilters,
+    dataSource,
+    setFilters,
+    isLoading: isDataSourceLoading,
+  } = useDataSource<tParsedIndexPattern, PatternDataSource>({
+    DataSource: StatisticsDataSource,
+    repository: new StatisticsDataSourceRepository(),
+  });
+
+  const selectedNodeFilter = {
+    meta: {
+      removable: false,
+      index: dataSource?.id,
+      negate: false,
+      disabled: false,
+      alias: null,
+      type: 'phrase',
+      key: null,
+      value: null,
+      params: {
+        query: null,
+        type: 'phrase',
+      },
+    },
+    query: {
+      match: {
+        nodeName: clusterNodeSelected,
+      },
+    },
+    $state: {
+      store: 'appState',
+    },
+  };
```

Comment:
**nitpick**: it is created in each render and could not be used on manager mode.

Reply:
A new DashboardTabsPanels component is created that abstracts the common behavior between DashboardListenerEngineStatistics and DashboardAnalysisEngineStatistics